packages/cactus-plugin-ledger-connector-corda/src/test/typescript/integration/monitor-transactions-v4.8.test.ts

```diff
@@ -0,0 +1,760 @@
+/**
+ * Test low-level state change monitoring interface in Kotlin Corda v4 component.
+ */
+
+// Contants: Log Levels
+const testLogLevel: LogLevelDesc = "debug";
+const sutLogLevel: LogLevelDesc = "info";
+
+// Contants: Test ledger
+const ledgerImageName = "ghcr.io/outsh/cactus-corda-4-8-all-in-one-obligation";
```

Comment:
@petermetz Please upload image from this branch to cactus ghcr, and I will replace this reference.